common/lc_library.cpp

```diff
@@ -1086,7 +1128,8 @@ bool lcPiecesLibrary::SaveCachePiece(PieceInfo* Info)
 {
 	lcMemFile MeshData;
 
-	const qint32 Flags = mStudLogo;
+	const qint32 Flags = mStudLogo | mLegoStyleDisplay;
```

Comment:
mStudLogo can be 0-5 so you can't or both directly

Reply:
Indeed, I should have caught that.

Cheers,